src/plugins/plugin.title.js

```diff
@@ -119,12 +118,8 @@ class Title extends Element {
 		}
 
 		lineCount = helpers.isArray(opts.text) ? opts.text.length : 1;
-		if (me._isPaddingObj()) {
-			padding = opts.padding.bottom + opts.padding.top;
-		} else {
-			padding = opts.padding * 2;
-		}
-		textSize = lineCount * helpers.options._parseFont(opts).lineHeight + padding;
+		opts.padding = helpers.options.toPadding(opts.padding);
```

Comment:
Should not mutate the options, use local variable for the result.
`me._padding` for example